realm/userposts.gno

```diff
@@ -12,11 +11,10 @@ import (
 // A user's "userPosts" may contain other posts (from followed users, etc.) but this only
 // has the top-level posts from the user (not replies to other posts).
 type UserPosts struct {
-	url       string
-	userAddr  std.Address
-	threads   avl.Tree // Post.id -> *Post
-	postsCtr  uint64   // increments Post.id
-	createdAt time.Time
```

Comment:
Why do you remove the `createAt` field?

Reply:
This is the time that the top-level UserPosts object is created (not the time of an individual post). This field is not used anywhere. This was copy/paste from the boards realm, in that case it is the time when the board is created. (Not very interesting.) The boards realm was not using this field either.